phys/module_sf_noahmplsm.F

```diff
@@ -4333,7 +4364,8 @@ SUBROUTINE RAGRB(parameters,ITER   ,VAI    ,RHOAIR ,HG     ,TAH    , & !in
 
        TMPRB  = CWPC*50. / (1. - EXP(-CWPC/2.))
        RB     = TMPRB * SQRT(parameters%DLEAF/UC)
-       RB     = MAX(RB,100.0)
+       RB     = MIN(MAX(RB, 5.0),50.0) ! limit RB to 5~50
+!       RB     = MAX(RB,100.0)  ! typically RB<50,so this leads to constant RB of 100
 !       RB = 200
```

Comment:
same idea, would it be better to remove old code that is not physically reasonable?

Reply:
will do.